sdlf-cicd/nested-stacks/template-cicd-cfn-module.yaml

```diff
@@ -167,6 +172,7 @@ Resources:
                     && unzip -q aws-sam-cli-linux-x86_64.zip -d sam-installation
                     ./sam-installation/install \
                     && sam --version
+                - pip3 install cfn-lint==0.87.7
```

Comment:
remove this line as it has been updated in `main` and would cause a conflict: f2fe2f8a2178ed7997724062719666890e6a5fe8

one thing you could do is cherry-pick f2fe2f8a2178ed7997724062719666890e6a5fe8 or rebase on top of main, although you would need to resync the main branch of your fork repository